api/v1alpha1/repository_types.go

```diff
@@ -85,7 +87,8 @@ type RepositorySpec struct {
 
 	// RepositoryAuth if the chart repository requires auth authentication,
 	// set the username and password to secret, with the fields user and password respectively.
-	RepositoryAuth string `json:"repositoryAuth,omitempty"`
+	RepositoryAuth      string `json:"repositoryAuth,omitempty"`
```

Comment:
I guess this would be 
```yaml
auth: // or authSecret?
  name: xxx
  namespace: xxx
```

Reply:
`RepositoryAuth` is the name of `secret`.

Reply:
secret should be in same namespace as the repository

Reply:
As we discussed, all CRDs should be  namespaced scope.